modules/rosa-autoscaling-nodes.adoc

```diff
@@ -1,15 +1,18 @@
 
-
-
 // Module included in the following assemblies:
 //
 // rosa-nodes/rosa-autoscale-nodes.adoc
 
 [id="rosa-autoscaling_{context}"]
-= Autoscaling nodes in a cluster
+= Autoscaling nodes in an existing cluster
 
 
-Configure autoscaling to dynamically scale the number of worker nodes up or down based on load. Autoscaling for worker nodes is configured in the machine pool definition.
+Configure autoscaling to dynamically scale the number of worker nodes up or down based on load. Autoscaling for worker nodes is configured in the machine pool definition if you are configuring an existing cluster.
+
+[NOTE]
```

Comment:
I think that this note might be more helpful in the parent assembly with a link to where you'd configure this during installation, if it's available.

Reply:
Done.